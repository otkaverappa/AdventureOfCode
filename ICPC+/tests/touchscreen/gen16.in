19
dblxhikpvflcrgnw 10
wlxdsgcjhyafmnhp
oupveznzgydaerwc
cwfucjflkhtwuank
wchbeucmugmyzlcb
jjxltexdlqbhsotq
tcrzxwnrccsdnwgw
gfjzjiewaffsvajq
ccpbafvehphulnsr
sdqdmwbmdghbiqrl
ujovojzvyhrkwlbp
quucswrweyxnqqak 10
bohryjoyshkosmfj
icnayeyeextuqthr
iqjizxirftizhpkp
rarreswlppfhlmzv
dkfehnyogionxaer
ayihqgshyzpjoogt
ylxhyxyhfoudobwr
zeapnsyltnwhdcad
nannxludzripuegt
kgkzaikuyidbkehz
gupdhligcqxwwerj 10
mbkpmxjkfonsuwta
siezuohyhfxflqox
sbmgyxqfmfxgcrju
bnwxddwkivqtlgtf
hflffdltjkznbkjf
zfcdjantvdpjkior
pazwfmpowpdabnfb
sjedkrzfvqqhafap
hzmmlddkugkwtrxo
abtmssspikwipxay
yonmrqwmwiisbfib 10
iboauiqesopklpkm
fxyynummeugfboij
pymlieqdsfngwxsd
xqbkmoyqkevoufxk
elxmpnpiueoseixb
ybnmpocbszppepbi
ayxsooaktqdzaccy
dpksdowynnntcqef
obxcrzpkrulrwnra
dduirqigexzinfpc
imganxmeryxolooq 10
tibmbjshhuruzhyi
thkjewpwwnkidaaz
kdllneuwamqztrjo
atzespcoeoyjobiz
eumrajnavgbqznez
iedavgpzupklqtmv
pyophcrekuujhbiq
hnscvjbszldpgrmv
pbmzfffpacajdkbk
zumvdppccuullgic
jvdocjfelfoorpar 10
lmopddtfzpqkyzmh
wswabbfojteckevw
sjnvoicpyvzwwodu
gbujfzzqudshjofb
zsyqacfazhwvvcpd
dmoknnaksvrblyek
qecsgitgrrdovvtz
hhluwmgohzrsxvep
bhihrdpiwtysorrx
yetwqamzzdrzayob
fykybciyvgqlxjkx 10
ndtgfffhkzgmxupe
vbcwdkwaqonpzxmp
dhvkmcrybzmywbcr
cgngqmiiawxavjry
qmjeocescqsytxpw
deewqmetkevgnmeg
apmosriuhbsbyjzd
nfzgrgzckwiziofi
esyylhsukmxkvwnk
bpqvxrzhnjixzqhd
kgdvpvpbjplenarp 10
rjmobnxqwgqvyxbi
feducuxljisxljoc
uctxpqonygiwdlel
sjhwgfhrnboamedh
gyexoukpcvngiuta
dawlfgdvjtvwyafg
yjepgoeiltqunjut
kterbhmkbkibknhl
xnafeenpzgjnpggb
zmtcuhmxrvyeihph
vpozvcoukahzgpbh 10
cwjydvvxsvbdeqmb
fbacfrwptgozvrgz
prxvoushrtmwlyxs
zauetqvmwmnudvtv
mrqanijfdvdquciv
ecaxsxlrjalpvgkj
zamnkvuprxfnznie
pkdjircrrngpwqav
smieievbbaodpwjg
josrfuizkpqghqbc
fmipqequggzyeien 10
ywegtpfdgxjppmru
aajrgbnmhmkowrbu
piaizhnfgyuwloql
qcewdrkndxdzogwd
qwopgbwmatinhbaz
ffxkyhzeecfujbaz
aorgrnurgdgpghol
onvmvvqbzwwkzwmz
mdhdscxzhdqnmhaa
uxpruhtufrefnqgz
voeoqbpzhhmtoowl 10
llcguxcaoghdznew
bjktmbstjgnzujki
wpoqmqqdyxgznnwq
wimikgetnrujcfry
uhqizjlaiuzvhxmf
gapsgtnvmhfqmyoi
ihtjqejyalwjikqq
kiiqdvmqdtgrtxad
etnwzwucjqlucekm
mvdpqrhvmonhnplt
kyplwlngebagfmtt 10
hyiarqvfgkmvbxon
xfzuqoawrcdyoyrw
ycwrutwcgkxhilxf
qyziobfgdkgsiaqi
cozyhxaphxzpiwxa
uyiiapqfzxzjzprb
dsbnpbcxyboiznkt
mueokwvlvvvukoxq
jydyzhxzlojkbtgp
okfygblcwgwixwyg
udewnevysfitaojo 10
yoohralpijyffyla
eswrwtpqyzjzoupo
lexcgisousuzshbw
bxnzteptebusvkii
qgnwqhkkzgkrolqp
lfrekgzqhukfespw
ycsplezmllgbzwsk
bjqnqpdzlnergung
yfxkmzwxkcylaqxc
aprsfvrsiwkqsxzs
fwervdoihotietmg 10
idypaqilosbipccw
zhnwkdgtrcbyvped
tfttxbenvivkkxhl
gwisaomrsppnfwra
dktanapikkuujdgs
aqkcgytzolmvjfvo
rqoeqfocpjxaofuq
xfsdfoevztqkylyq
cowutnxkywmmdhda
oxftlkpmdhybvwtx
msthfsrdogrtnwtd 10
uzwflnuoxsrsrnrd
hknmfgrvmipbhieb
jdiwsempxghotatc
ngosnhnzrcbynhby
kkvdojunpedkexmr
dcjsjzubbxbqedpr
pmwfvqtnwwxbvkub
mgtyfnbimczrhqkw
egebyxovvmwryssm
ynkddnmpqnizfsyj
ackzzbuwnsplkjzj 10
xkmczatrperuypfz
rsystuqimgvwpufo
guqgwkxmqogodonw
gnqzhhhtnescyzrf
tjlqvielzmceapbg
fthoaqkpucstblax
ulpsvtduffzgxaoe
vwswoeljgfeksehp
ryhmrmhyuierkvxf
trbhwpscuwoncxcv
vlioxrnrztkjohoh 10
zspvhjzdiosknugj
fqxfhmwighruqhcp
ztmgcmmkcexpzfag
xylekkostfnjmrzo
klwozibbnarofrwc
rkieuwwndlyrcxfo
lefmmgozihpnamru
ybauzyjcmhvohdfu
hkgtswvafmpgaicy
jeskcboojjfsmmmv
ysrqpmswyjebrgzc 10
mumpvcfglmaaamvy
foowaisbrxckfdms
xajvcqdqfeqfqnex
cuvedqhwplhuptoo
wxlyqpqvticlyglc
dgggwqelblishwjd
wudojwjeenscudex
jnfidlvgxdzgckla
eooqmzurmotiryhd
lolqaiwzmxfqhqqm
ggetfyktpdcgdljq 10
awiaffztcgjmwbac
hevoefjtllaqxljz
irbnycidirpeurgd
vetzlcvyqxppiaos
tphsrpxzimfcflfa
pabbdyztvqiesyym
pgggvdhermgwzowr
ozutyvmvowzguasl
gyrddahuoqqogohw
qbqowejkblrxlljt
