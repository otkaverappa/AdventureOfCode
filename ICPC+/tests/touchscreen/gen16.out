gfjzjiewaffsvajq 42
tcrzxwnrccsdnwgw 53
wchbeucmugmyzlcb 57
cwfucjflkhtwuank 59
sdqdmwbmdghbiqrl 61
ccpbafvehphulnsr 64
ujovojzvyhrkwlbp 66
wlxdsgcjhyafmnhp 67
oupveznzgydaerwc 68
jjxltexdlqbhsotq 76
dkfehnyogionxaer 56
icnayeyeextuqthr 56
zeapnsyltnwhdcad 57
bohryjoyshkosmfj 62
kgkzaikuyidbkehz 63
ayihqgshyzpjoogt 67
rarreswlppfhlmzv 67
nannxludzripuegt 70
ylxhyxyhfoudobwr 70
iqjizxirftizhpkp 72
sjedkrzfvqqhafap 50
sbmgyxqfmfxgcrju 52
siezuohyhfxflqox 54
hflffdltjkznbkjf 57
mbkpmxjkfonsuwta 61
bnwxddwkivqtlgtf 65
hzmmlddkugkwtrxo 66
pazwfmpowpdabnfb 68
zfcdjantvdpjkior 73
abtmssspikwipxay 82
fxyynummeugfboij 51
obxcrzpkrulrwnra 54
pymlieqdsfngwxsd 54
dduirqigexzinfpc 56
dpksdowynnntcqef 58
elxmpnpiueoseixb 60
iboauiqesopklpkm 62
ybnmpocbszppepbi 67
ayxsooaktqdzaccy 71
xqbkmoyqkevoufxk 84
hnscvjbszldpgrmv 52
pyophcrekuujhbiq 53
iedavgpzupklqtmv 60
pbmzfffpacajdkbk 61
zumvdppccuullgic 61
tibmbjshhuruzhyi 64
kdllneuwamqztrjo 65
thkjewpwwnkidaaz 65
eumrajnavgbqznez 67
atzespcoeoyjobiz 68
gbujfzzqudshjofb 53
qecsgitgrrdovvtz 63
hhluwmgohzrsxvep 64
dmoknnaksvrblyek 67
wswabbfojteckevw 67
sjnvoicpyvzwwodu 69
lmopddtfzpqkyzmh 71
bhihrdpiwtysorrx 75
yetwqamzzdrzayob 78
zsyqacfazhwvvcpd 81
apmosriuhbsbyjzd 47
ndtgfffhkzgmxupe 48
dhvkmcrybzmywbcr 49
cgngqmiiawxavjry 53
qmjeocescqsytxpw 55
esyylhsukmxkvwnk 57
deewqmetkevgnmeg 62
bpqvxrzhnjixzqhd 65
vbcwdkwaqonpzxmp 73
nfzgrgzckwiziofi 84
uctxpqonygiwdlel 43
kterbhmkbkibknhl 49
sjhwgfhrnboamedh 54
dawlfgdvjtvwyafg 56
zmtcuhmxrvyeihph 65
gyexoukpcvngiuta 66
yjepgoeiltqunjut 73
xnafeenpzgjnpggb 74
feducuxljisxljoc 77
rjmobnxqwgqvyxbi 77
ecaxsxlrjalpvgkj 58
fbacfrwptgozvrgz 60
smieievbbaodpwjg 61
cwjydvvxsvbdeqmb 67
zamnkvuprxfnznie 67
mrqanijfdvdquciv 68
josrfuizkpqghqbc 69
zauetqvmwmnudvtv 73
prxvoushrtmwlyxs 79
pkdjircrrngpwqav 89
qwopgbwmatinhbaz 57
ffxkyhzeecfujbaz 59
mdhdscxzhdqnmhaa 60
aorgrnurgdgpghol 61
uxpruhtufrefnqgz 61
aajrgbnmhmkowrbu 63
piaizhnfgyuwloql 65
onvmvvqbzwwkzwmz 66
qcewdrkndxdzogwd 67
ywegtpfdgxjppmru 70
mvdpqrhvmonhnplt 51
uhqizjlaiuzvhxmf 59
bjktmbstjgnzujki 60
kiiqdvmqdtgrtxad 60
ihtjqejyalwjikqq 61
wimikgetnrujcfry 63
etnwzwucjqlucekm 67
llcguxcaoghdznew 67
gapsgtnvmhfqmyoi 68
wpoqmqqdyxgznnwq 79
uyiiapqfzxzjzprb 40
okfygblcwgwixwyg 52
hyiarqvfgkmvbxon 56
xfzuqoawrcdyoyrw 56
mueokwvlvvvukoxq 60
jydyzhxzlojkbtgp 62
dsbnpbcxyboiznkt 66
ycwrutwcgkxhilxf 66
qyziobfgdkgsiaqi 67
cozyhxaphxzpiwxa 85
bxnzteptebusvkii 44
lexcgisousuzshbw 53
lfrekgzqhukfespw 53
qgnwqhkkzgkrolqp 55
ycsplezmllgbzwsk 55
eswrwtpqyzjzoupo 61
yfxkmzwxkcylaqxc 64
yoohralpijyffyla 67
bjqnqpdzlnergung 68
aprsfvrsiwkqsxzs 70
dktanapikkuujdgs 40
tfttxbenvivkkxhl 48
oxftlkpmdhybvwtx 53
gwisaomrsppnfwra 56
cowutnxkywmmdhda 57
rqoeqfocpjxaofuq 58
idypaqilosbipccw 59
zhnwkdgtrcbyvped 60
xfsdfoevztqkylyq 61
aqkcgytzolmvjfvo 62
mgtyfnbimczrhqkw 39
hknmfgrvmipbhieb 46
egebyxovvmwryssm 49
jdiwsempxghotatc 49
ngosnhnzrcbynhby 49
kkvdojunpedkexmr 55
pmwfvqtnwwxbvkub 55
dcjsjzubbxbqedpr 56
uzwflnuoxsrsrnrd 56
ynkddnmpqnizfsyj 63
xkmczatrperuypfz 56
ryhmrmhyuierkvxf 61
gnqzhhhtnescyzrf 64
rsystuqimgvwpufo 64
trbhwpscuwoncxcv 64
fthoaqkpucstblax 68
vwswoeljgfeksehp 72
tjlqvielzmceapbg 77
ulpsvtduffzgxaoe 82
guqgwkxmqogodonw 85
hkgtswvafmpgaicy 55
klwozibbnarofrwc 58
ybauzyjcmhvohdfu 58
ztmgcmmkcexpzfag 65
jeskcboojjfsmmmv 66
rkieuwwndlyrcxfo 66
xylekkostfnjmrzo 66
zspvhjzdiosknugj 68
lefmmgozihpnamru 73
fqxfhmwighruqhcp 78
wudojwjeenscudex 56
eooqmzurmotiryhd 57
wxlyqpqvticlyglc 57
xajvcqdqfeqfqnex 57
foowaisbrxckfdms 60
jnfidlvgxdzgckla 66
lolqaiwzmxfqhqqm 66
dgggwqelblishwjd 69
mumpvcfglmaaamvy 69
cuvedqhwplhuptoo 74
gyrddahuoqqogohw 35
hevoefjtllaqxljz 37
pgggvdhermgwzowr 50
tphsrpxzimfcflfa 50
qbqowejkblrxlljt 53
irbnycidirpeurgd 57
ozutyvmvowzguasl 58
pabbdyztvqiesyym 61
awiaffztcgjmwbac 65
vetzlcvyqxppiaos 74
